packages/grafana-ui/src/components/EmptyState/EmptyState.tsx

```diff
@@ -43,7 +43,7 @@ export const EmptyState = ({
   children,
   image,
   message,
-  hideImage = false,
+  hideImage = true,
```

Comment:
This hides the Grafana image of a little goggle-wearing lizard/dinosaur thingie.